src/Pages/Templates/TemplateDetails/components/TemplateActionDropdown.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import {
+  Dropdown,
+  DropdownItem,
+  DropdownList,
+  MenuToggle,
+  MenuToggleElement,
+} from '@patternfly/react-core';
+import { useLocation, useNavigate, useParams } from 'react-router-dom';
+import { useQueryClient } from 'react-query';
+import { useDeleteTemplateItemMutate } from 'services/Templates/TemplateQueries';
+import { TEMPLATES_ROUTE } from 'Routes/constants';
+import ConditionalTooltip from 'components/ConditionalTooltip/ConditionalTooltip';
+import { useAppContext } from 'middleware/AppContext';
+
+export default function TemplateActionDropdown() {
+  const [isOpen, setIsOpen] = React.useState(false);
+  const { pathname } = useLocation();
+  const [mainRoute] = pathname?.split(`${TEMPLATES_ROUTE}/`) || [];
+  const baseRoute = mainRoute + `${TEMPLATES_ROUTE}/`;
+  const navigate = useNavigate();
+  const queryClient = useQueryClient();
+  const { templateUUID: uuid } = useParams();
+  const { rbac } = useAppContext();
+
+  const { mutateAsync: deleteItem, isLoading: isDeleting } =
+    useDeleteTemplateItemMutate(queryClient);
+
+  const onToggleClick = () => {
+    setIsOpen(!isOpen);
+  };
+
+  const onSelect = (_, value: string | number | undefined) => {
+    switch (value) {
+      case 'edit':
+        navigate(`${baseRoute}/${uuid}/edit`);
```

Comment:
same thing here as the tab routing, baseRoute ends with a `/` so the extra one between baseRoute and uuid breaks things